Sources/MusicXML/Complex Types/Note.swift

```diff
@@ -17,6 +17,12 @@ public struct Note {
 
     // MARK: - Attributes
 
+    public var position: Position = Position()
+    public var fontFamily: CommaSeparatedText?
+    public var fontStyle: FontStyle?
+    public var fontSize: FontSize?
+    public var fontWeight: FontWeight?
+    public var color: Color?
```

Comment:
Furthermore, can we wrap all these up in a `PrintStyle`?

Reply:
Of course that's a sure way to go.